code/__DEFINES/flags.dm

```diff
@@ -24,6 +24,7 @@
 #define NOSLIP		1024 		//prevents from slipping on wet floors, in space etc (NOTE: flag shared with THICKMATERIAL for external suits and helmet)
 
 #define OPENCONTAINER	4096	// is an open container for chemistry purposes
+#define SEMIOPENCONTAINER	8192

```

Comment:
and this is the _code_ not the game itself. The code must be as clear as possible (pun _intended_)